dylan-package.json

```diff
@@ -8,8 +8,8 @@
     "contact": "dylan-lang@googlegroups.com",
     "dependencies": [],
     "dev-dependencies": [
-	"testworks@3.2.0",
-	"sphinx-extensions@0.2.0"
+	"testworks",
```

Comment:
I agree that is less maintenance work, but it is a good practice? Is ok for developer dependencies only?

Reply:
Right, I only do it for dev deps.  (And my world is colored by the fact that we used to do everything at `master` all the time, so using package versions _at all_ is somewhat new in Dylandia.)